spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/AdminApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.data.rest;
```

Comment:
It is simply a redirect to the REST API right? The more important thing here is that the "-rest" suffix is mapping to the ".rest" package as the "namespace" of this project.